src/java.base/share/classes/java/io/InputStreamReader.java

```diff
@@ -50,9 +50,17 @@
  *     BufferedReader in = new BufferedReader(new InputStreamReader(anInputStream));
  * }
  *
+ * <P>To read from {@link System#in}, use the system property value
+ * {@link System##stdin.encoding stdin.encoding} as the {@code Charset}:
+ *
+ * {@snippet lang=java :
+ *     new InputStreamReader(System.in, System.getProperty("stdin.encoding"));
+ * }
+ *
```

Comment:
There are several places in the sources that still contain `new InputStreamReader(System.in)`. Shouldn't they be replaced with `new InputStreamReader(System.in, System.getProperty("stdin.encoding"))` too?)

Reply:
There are several locations in the jdk source (I simply grep'ed for `new InputStreamReader(System.in)`). And yes, they will need to be replaced with the recommended usage. I will address them with a different JBS issue.